backend/src/email/services/custom-domain.service.ts

```diff
@@ -143,9 +153,11 @@ const storeFromAddress = async (
  * 3. Checks the domain's dns to ensure that the cnames are there
  */
 const verifyFromAddress = async (email: string): Promise<void> => {
-  const dkimTokens = await verifyEmailWithAWS(email)
-
-  await verifyCnames(dkimTokens, email)
+  const [dkimTokensBackend, dkimTokensWorker] = await Promise.all([
+    verifyEmailWithAWS(email, backendSes, backendSesRegion),
+    verifyEmailWithAWS(email, workerSes, workerSesRegion),
+  ])
+  await verifyCnames([...dkimTokensBackend, ...dkimTokensWorker], email)
```

Comment:
We can refactor the `verifyCnames` function to use `Promise.all` as well. Do u wanna do it in this PR?

Reply:
good catch, done (not 100% sure, but think it's right)